constitution.tex

```diff
@@ -520,9 +520,11 @@
 \asubsubsection{The Introductory Process}
 The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
 \asubsubsubsection{The Evaluation Period}
-The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
+The Process will occur either once or twice per semester, and will last for six (6) weeks.
+The Process shall be initiated by the Evaluations Director during either the first or second week of each semester.
```

Comment:
With this wording, the introductory process can only begin for one week, either the first or second week of the semester. Some additional wording to allow packets to start for the first two weeks of the Introductory Process would be good.